app/containers/Chip/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Pressable, StyleSheet, View, Text, StyleProp, TextStyle, ViewStyle } from 'react-native';
+
+import { ISelectedUser } from '../../reducers/selectedUsers';
+import { useTheme } from '../../theme';
+import { CustomIcon, TIconsName } from '../CustomIcon';
+import sharedStyles from '../../views/Styles';
+import Avatar from '../Avatar';
+
+const styles = StyleSheet.create({
+	pressable: {
+		marginVertical: 16,
+		paddingHorizontal: 8,
+		marginRight: 8,
+		borderRadius: 2,
+		height: 40,
+		justifyContent: 'center',
+		maxWidth: 192
+	},
+	container: {
+		flexDirection: 'row',
+		alignItems: 'center'
+	},
+	avatar: {
+		marginRight: 8,
+		marginVertical: 8
+	},
+	textContainer: {
+		marginRight: 8,
+		maxWidth: 120
+	},
+	name: {
+		fontSize: 16,
+		...sharedStyles.textMedium
+	}
+});
+
+export interface IChip {
+	avatar?: string;
+	text: string;
+	item: ISelectedUser;
+	onPress?: Function;
+	testID?: string;
+	iconName?: TIconsName;
+	style?: StyleProp<ViewStyle>;
+	textStyle?: StyleProp<TextStyle>;
+}
+
+const Chip = ({ item, avatar, text, onPress = () => {}, testID, iconName, style, textStyle }: IChip) => {
+	const { colors } = useTheme();
+
+	return (
+		<Pressable
+			testID={testID}
+			style={({ pressed }) => [
+				styles.pressable,
+				{
+					backgroundColor: pressed ? colors.bannerBackground : colors.auxiliaryBackground
+				},
+				style
+			]}
+			onPress={() => onPress(item)}
```

Comment:
It should be optional, so we might not have it.
```suggestion
			onPress={() => onPress?.()}
```

`Pressable` shouldn't have active state if they don't have `onPress`.
They should behave just like a `View`.

Make sure to test the existence of the `onPress` on unit tests.